presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -121,6 +125,53 @@ public void testPublic()
         assertThat(aliceExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasNoRows();
     }
 
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testTableOwnerPrivileges()
+    {
+        String tableName = "alice_table";
```

Comment:
However, tests shouldn't leave side effects on the cluster. If you need to debug a test, you can easily re-run the test. The AlterTableTests use try/finally, as do the QG tests.